### PR TITLE
Fall 2019 recalibration

### PR DESCRIPTION
**Pull request by Gregg140**
Fall 2019 re-calibration.

### CODE REVIEW

**Comment by jeanconn** [comment]
I think for this release the plan will be to have the json files match between this ACIS dpa model and the Matlab 2019_207 release.  @matthewdahmer do you agree the file in this branch is what you expect?

**Comment by matthewdahmer** [comment]
Yes, I have confirmed the correct file is used in the FOT Tools, indicated by a commit hash of 41a9ff3.

File: dpa_check/Release_Notes_V2.5.0.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Presented to and approved by TWG.
+
+
+
+Deployment Plan
+===============
+
+Deploy after the first segment of the load presently being reviewed, 
+starts execution.
```
Comment (jeanconn) [inline dpa_check/Release_Notes_V2.5.0.txt]: I'm not sure how this bit applies.

File: dpa_check/Release_Notes_V2.5.0.txt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Reviewed by ACIS Ops.
+
+Presented to and approved by TWG.
+
+
+
+Deployment Plan
+===============
+
+Deploy after a new FOT Tools release has FDB approval. Will be running
```
Comment (jeanconn) [inline dpa_check/Release_Notes_V2.5.0.txt]: OK.  If the plan is to deploy after 2019_207 is promoted, this looks good.

**Comment by jeanconn** [comment]
I think this is good to merge and tag.

**Comment by jzuhone** [comment]
I’ll handle this.

**Comment by jeanconn** [comment]
With regard to merge and tag, I think we can move to a plan where you merge and tag after github approval without waiting for LR approval.  

My point is just that the sooner you merge and tag the sooner it is convenient for us to actually build a dpa_check package and have it ready.

**Comment by jzuhone** [comment]
@jeanconn sure thing—I’m on the road at the moment but I plan to tag here in a little bit

**Pull request merged**